packages/manager/modules/adp/src/deploy/deploy.scss

```diff
@@ -0,0 +1,52 @@
+// override bootstrap map to add custom widths
+// This variable affects the `.h-*` and `.w-*` classes.
+$sizes: () !default;
+// stylelint-disable-next-line scss/dollar-variable-default
+$sizes: map-merge(
+  (
+    10: 10%,
```

Comment:
I'm a bit curious about why need to provide more sizes' values.

Could you tell us in which context you have to uses theses extra helpers?

Reply:
The additional helpers are currently not being used. Hence they have been removed